plugins/chat/spec/system/drawer_spec.rb

```diff
@@ -80,7 +80,7 @@
       drawer.open_channel(channel_1)
       channel_page.hover_message(message_1)
 
-      expect(page).to have_css(".chat-message-actions-container")
+      expect(page).to have_css(".chat-message-actions-container", visible: :all)
```

Comment:
Due to the hide fix above, it's now possible for the container to be present but not visible.